public/ballers_new.sql

```diff
@@ -0,0 +1,164 @@
+-- phpMyAdmin SQL Dump
+-- version 5.0.1
+-- https://www.phpmyadmin.net/
+--
+-- Host: localhost
+-- Generation Time: Jun 02, 2020 at 07:43 PM
+-- Server version: 10.4.11-MariaDB
+-- PHP Version: 7.4.2
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET AUTOCOMMIT = 0;
+START TRANSACTION;
+SET time_zone = "+00:00";
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+
+--
+-- Database: `ballers_new`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `area`
+--
+
+CREATE TABLE `area` (
+  `area_id` int(255) NOT NULL,
+  `area_name` text NOT NULL,
+  `state_id` int(255) NOT NULL,
+  `state_name` varchar(255) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+--
+-- Dumping data for table `area`
+--
+
+INSERT INTO `area` (`area_id`, `area_name`, `state_id`, `state_name`) VALUES
+(1, 'Lekki Phase 1', 1, 'Lagos'),
+(2, 'Orchid Hotel Road', 1, 'Lagos'),
+(3, 'Wuse II', 2, 'Abuja');
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `house-type`
+--
+
+CREATE TABLE `house-type` (
+  `house_id` int(255) NOT NULL,
+  `type` varchar(255) NOT NULL,
+  `price` varchar(255) NOT NULL,
+  `area_id` int(255) NOT NULL,
+  `area_name` varchar(255) NOT NULL,
+  `state_id` int(255) NOT NULL,
+  `state_name` varchar(255) NOT NULL
```

Comment:
The state_name and area_name are not necessary and should be removed since it can be gotten from the state table and area table respectively.